lib/auth/widgets/custom_button.dart

```diff
@@ -0,0 +1,39 @@
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+import 'package:google_fonts/google_fonts.dart';
+
+class CustomButton extends StatelessWidget {
+  final VoidCallback onPressed;
+  final String buttonText;
+  final double fontSize;
```

Comment:
Should we use an already made widget for this? (/auth/widgets/cusom_button.dart)